app/controllers/api/v2/bioscan/export_pool_xp_to_traction_controller.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    module Bioscan
+      # Endpoint to export a PoolXP tube to Traction
+      class ExportPoolXpToTractionController < ApplicationController
+        before_action :login_required, except: [:create]
+        skip_before_action :verify_authenticity_token
+
+        def create
+          Rails.logger.debug "DEBUG: In ExportPoolXpToTractionController create"
+
+          errors = preflight_errors(barcode)
+          render json: { errors: errors }, status: :unprocessable_entity and return unless errors.empty?
+
+          Delayed::Job.enqueue ExportPoolXpToTractionJob.new(barcode)
+          render json: {}, status: :ok
+        end
+
+        private
+
+        def preflight_errors(barcode)
+          # Check that the tube exists
+          tube = Tube.with_barcode(barcode).first
```

Comment:
Not important, but again could change to the following

```suggestion
          tube = Tube.find_by_barcode(barcode)
```